front/src/Components/Chat/ChatElement.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+    import {ChatMessageTypes} from "../../Stores/ChatStore";
+    import type {ChatMessage} from "../../Stores/ChatStore";
+    import {HtmlUtils} from "../../WebRtc/HtmlUtils";
+
+    export let message: ChatMessage
+
+    $: urlifyText = HtmlUtils.urlify(message.text);
+</script>
+
+<p>
+{#if message.type === ChatMessageTypes.text}
+    <b>{message.author}:</b> <span 	contenteditable="true" bind:innerHTML={urlifyText}></span>
```

Comment:
wasnt there a condition or changing the current user to 'Me' ?

Reply:
You're right, I added a special case for 'me'